rollup/core/classes/RollupParentResetProcessor.cls

```diff
@@ -59,6 +70,11 @@ public without sharing class RollupParentResetProcessor extends RollupFullBatchR
       // down the rabbit hole we go again
       processId = this.startAsyncWork();
     } else if (this.countOfItems > 0) {
+      for (RollupAsyncProcessor proc : this.rollups) {
+        isRecursiveRun = true;
+        proc.runCalc();
+        isRecursiveRun = false;
+      }
```

Comment:
Could you instead set `isRecursiveRun` outside of the `for` loop (before and after)? Right now, it looks like it'd be flipped to `false` at the end of a `proc`, then immediately flipped back to `true` for the next `proc`.

```suggestion
      isRecursiveRun = true;
      for (RollupAsyncProcessor proc : this.rollups) {
        proc.runCalc();
      }
      isRecursiveRun = false;
```

Reply:
Ah yeah, that makes perfect sense